### ISSUE TITLE
Add maxima of datasets into maximum along contour plot

### ISSUE THREAD

**Issue by ahaselsteiner** [ec-benchmark-organizers/ec-benchmark#8]
@ryancoe If that is okay with you I'd add the maxima of the datasets into the plot that shows the maxima along the contour.
They are of course not directly comparable, but offer an interesting reference point.

Old:

![image](https://user-images.githubusercontent.com/11452055/94161632-2cdd6100-fe86-11ea-8b78-31d9b4017cde.png)

New:

![image](https://user-images.githubusercontent.com/11452055/94161380-e6880200-fe85-11ea-8ffd-ac72f4c39508.png)

**Comment by rgcoe** [comment]
Yes, great idea!

**Comment by ahaselsteiner** [comment]
Okay, thanks, I added it into the manuscript.

I'll also create such plot with the 1-year contours and I'll plot the the 1-year empirical quantile for reference.

**Comment by ahaselsteiner** [comment]
For now, I plotted the maxima along the 1-year contour, but did not add the empirical 1-year quantile as we would need an extra legend and some extra explanation.

Here is the plot: 

![image](https://user-images.githubusercontent.com/11452055/94275863-c751a900-ff47-11ea-8cf4-5d29057dd39c.png)

**Comment by rgcoe** [comment]
Looks very nice @ahaselsteiner! Can you post the version with the quantiles here. I agree it may not be worth it to include, but I'm curious to see these quantiles nonetheless.

**Comment by ahaselsteiner** [comment]
@ryancoe Yes, see below. Actually, now having plotted it I find it an interesting reference.

Another thing that I changed: I am now using two different marker for the two broad different kind of contours (contours based on marginal exceedance [IFORM, direct sampling, direct IFORM; circle] versus contours based on total exceedance [ISORM, highest density, inverse direct simulation; triangle]).

Total exceedance contours should have higher maxima along the contours than marginal exceedance contours. The 1-yr marginal exceedance contours should have a maximum hs along the contour that is about the same as the empirical marginal 1-yr return value.

![image](https://user-images.githubusercontent.com/11452055/94350160-da4da180-004b-11eb-9614-e4c6e5f03fd5.png)

I'll put this updated figure into overleaf - let me ne know in case you think the quantiles are confusing as I first worried and should not be plotted.
